crates/starknet-devnet/tests/common/background_anvil.rs

```diff
@@ -116,13 +121,13 @@ impl BackgroundAnvil {
             .await
             .map_err(|e| {
                 TestError::EthersError(format!(
-                    "tx for deposit l1l2 contract on ethereum failed: {e}"
+                    "tx for withdraw l1l2 contract on ethereum failed: {e}"
                 ))
             })?
             .await
             .map_err(|e| {
                 TestError::EthersError(format!(
-                    "tx for deposit l1l2 contract on ethereum no receipt: {e}"
+                    "tx for withdraw l1l2 contract on ethereum has no receipt: {e}"
```

Comment:
```suggestion
                    "tx for withdraw l1-l2 contract on ethereum has no receipt: {e}"
```

Reply:
Addressed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/597/commits/b5040ca8266c91e5b4ab06979c8da4c318f40773